### PR TITLE
Use separate model for send message real-time notification

### PR DESCRIPTION
**Pull request by kolosovpetro**
Fixes: https://github.com/MangoInstantMessenger/MangoMessengerAPI/issues/449

### CODE REVIEW

File: MangoAPI.BusinessLogic/ApiCommands/Messages/SendMessageCommandHandler.cs [hunk 1]
```diff
@@ -91,20 +91,34 @@ public class SendMessageCommandHandler
...
 
-        var authorPictureUrl = $"{blobServiceSettings.MangoBlobAccess}/{user.Image}";
+        var authorImageUrl = $"{blobServiceSettings.MangoBlobAccess}/{user.Image}";
 
         var attachmentUrl = attachmentUniqueFileName == null
             ? null
             : $"{blobServiceSettings.MangoBlobAccess}/{attachmentUniqueFileName}";
 
-        var messageResultDto = messageEntity.ToMessage(
+        // var messageNotification = messageEntity.ToMessage(
```
Comment (kolosovpetro) [inline MangoAPI.BusinessLogic/ApiCommands/Messages/SendMessageCommandHandler.cs:100]: Remove commented code

File: MangoAPI.BusinessLogic/Models/Message.cs [hunk 2]
```diff
@@ -42,33 +41,33 @@ public record Message
...
-            InReplyToText = message.InReplyToText,
-            AuthorImageUrl = authorPictureUrl,
-            AttachmentUrl = attachmentUrl
-        };
-
-        return messageDto;
-    }
-}
\ No newline at end of file
+// public static class MessageMapper
```
Comment (kolosovpetro) [inline MangoAPI.BusinessLogic/Models/Message.cs:44]: Remove commented code

File: MangoAPI.BusinessLogic/Notifications/MessageDeleteNotification.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace MangoAPI.BusinessLogic.Notifications;
+
+// public record MessageDeleteNotification
```
Comment (kolosovpetro) [inline MangoAPI.BusinessLogic/Notifications/MessageDeleteNotification.cs:5]: Remove commented code

File: MangoAPI.Client/src/app/components/chats/chats.component.ts [hunk 4]
```diff
@@ -397,54 +441,62 @@ export class ChatsComponent implements OnInit {
...
 
-    if (notification.isLastMessage) {
-      this.activeChat.lastMessageText = notification.modifiedText;
-      this.activeChat.lastMessageTime = notification.updatedAt;
-    }
+    return message;
   }
 
   private onMessageDeleteHandler(notification: DeleteMessageNotification) {
+    console.log('delete message notification: ' + JSON.stringify(notification));
```
Comment (kolosovpetro) [inline MangoAPI.Client/src/app/components/chats/chats.component.ts:493]: Remove console log

File: MangoAPI.Client/src/app/components/chats/chats.component.ts [hunk 5]
```diff
@@ -522,9 +576,13 @@ export class ChatsComponent implements OnInit {
     this.messageText += emoji;
   }
 
-  chatContainsMessages(chat: Chat): boolean {
-    return chat.lastMessageAuthor != null && chat.lastMessageText != null;
-  }
+  // chatContainsMessages(chat: Chat): boolean {
```
Comment (kolosovpetro) [inline MangoAPI.Client/src/app/components/chats/chats.component.ts:579]: Remove commented code

File: MangoAPI.Client/src/app/components/chats/chatHasMessagesPipe.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Chat } from '../../types/models/Chat';
+
+@Pipe({ name: 'chatHasMessages' })
+export class ChatHasMessagesPipe implements PipeTransform {
+  transform(value: Chat): boolean {
+    console.log(JSON.stringify(value));
```
Comment (kolosovpetro) [inline MangoAPI.Client/src/app/components/chats/chatHasMessagesPipe.ts:7]: Remove console log

File: MangoAPI.BusinessLogic/Models/Message.cs [hunk 7]
```diff
@@ -42,33 +41,33 @@ public record Message
...
+//             Self = message.UserId == userId,
+//             InReplyToUser = message.InReplyToUser,
+//             InReplyToText = message.InReplyToText,
+//             AuthorImageUrl = authorPictureUrl,
+//             AttachmentUrl = attachmentUrl
+//         };
+//
+//         return messageDto;
+//     }
+// }
```
Comment (Ketteiteki) [inline MangoAPI.BusinessLogic/Models/Message.cs:73]: remove comments

**Review by Ketteiteki — APPROVED** [review]

**Review by Ketteiteki — APPROVED** [review]

**Pull request merged**